src/secrets/CommandWrite.ts

```diff
@@ -1,29 +1,24 @@
 import type PolykeyClient from 'polykey/dist/PolykeyClient';
-import * as errors from '../errors';
 import CommandPolykey from '../CommandPolykey';
+import * as binProcessors from '../utils/processors';
+import * as binParsers from '../utils/parsers';
 import * as binUtils from '../utils';
 import * as binOptions from '../utils/options';
-import * as binParsers from '../utils/parsers';
-import * as binProcessors from '../utils/processors';
 
-class CommandUpdate extends CommandPolykey {
+class CommandStat extends CommandPolykey {
   constructor(...args: ConstructorParameters<typeof CommandPolykey>) {
```

Comment:
`CommandStat` inside a `CommandWrite.ts` file? The class name should reflect the file name.

Reply:
Oops, I forgot to do this when I copied over the command boilerplate from another command. I will update this.